jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/server/FusekiServer.java

```diff
@@ -371,19 +367,6 @@ else if ( ! dir.isDirectory())
             throw new FusekiConfigException("Not a directory: "+directory) ;
     }
 
-    private static void mustExist(Path directory) {
```

Comment:
Please leave. While unused, these 3 methods form verbs for working with Paths - I'd prefer to leave the verbs around.